test/pj-rehearse-integration/run.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# This test sets up a temporary testing repository
+# resembling `openshift/release` and tries to execute
+# pj-rehearse as if run over a PR for that repo
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly WORKDIR="$( mktemp -d )"
+trap 'rm -rf "${WORKDIR}"' EXIT
+
+readonly TEST_ROOT="$( dirname "${BASH_SOURCE[0]}")"
+readonly FAKE_OPENSHIFT_RELEASE="${WORKDIR}/repo"
+
+make_testing_repository() {
+  local -r master_data="${PWD}/${TEST_ROOT}/master"
+  local -r candidate_data="${PWD}/${TEST_ROOT}/candidate"
+
+  echo "[INFO] Preparing fake input repository..."
+  mkdir "${FAKE_OPENSHIFT_RELEASE}"
+  pushd "${FAKE_OPENSHIFT_RELEASE}" >/dev/null
+  git init --quiet
+  cp -R "${master_data}"/* .
```

Comment:
I suspect `git clone`ing the original repository would be faster.

Reply:
I don't think we need to care about speed here, it's just a few files.